concepts/char/about.md

```diff
@@ -1,3 +1,42 @@
 # About
 
-TODO
+Rust implements the [char][char type] type to represent a single character. A `char` literal is placed within single quotes, like `'a'`.
+Each `char` is four bytes in size, which is large enough to represent most [Unicode][unicode] characters as [Unicode Scalar Values][unicode scalar].
```

Comment:
```suggestion
Each `char` is four bytes in size, which is large enough to represent [Unicode][unicode] characters as [Unicode Scalar Values][unicode scalar].
```

I'm not aware of any unicode codepoints which can't be represented as `char`.

Reply:
"The char type represents a single character. More specifically, since 'character' isn't a well-defined concept in Unicode, char is a 'Unicode scalar value', which is similar to, but not the same as, a 'Unicode code point'." 

"Unicode Scalar Value. Any Unicode code point _except high-surrogate and low-surrogate code points_. In other words, the ranges of integers 0 to D7FF16 and E00016 to 10FFFF16 inclusive."

https://doc.rust-lang.org/std/primitive.char.html
http://www.unicode.org/glossary/#unicode_scalar_value

Reply:
```suggestion
Each `char` is four bytes in size and represents a single [Unicode Scalar Values][unicode scalar].
```

I think this version is both well-defined and simpler; if unicode characters aren't a well-defined concept, let's just omit mentioning them.